services/rds/src/main/java/org/finra/gatekeeper/controllers/AuthController.java

```diff
@@ -66,4 +71,8 @@ public GetRoleResponseWrapper getRole(){
         }
         return result;
     }
+    @RequestMapping(value = "/loadUser", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
+    public Map<String, Set<GatekeeperADGroupEntry>> loadUser() {
+        return gatekeeperRoleService.getRestrictedRoleMemberships();
```

Comment:
Can we include this as part of getRole() so that we don't need an additional http call? You can add it into the GetRoleResponseWrapper class

Reply:
I guess this is for debug?